### PR TITLE
OPENEUROPA-1807: featured_image must be after content_moderation_control

### PR DESCRIPTION
**Pull request by dxvargas**
### Description

Change weight for `featured_image` to appear after `content_moderation_control`.

### Change log

- Changed: Weight for `featured_image` decreased from -100 to -19.

### CODE REVIEW

**Comment by dxvargas** [comment]
@upchuk weights are always relative. We always need to worry about what we have in other places.
Even if using -1000 someone can put in other place (also in `featured_image`) weights of -9999.
It's up to each one's imagination.

Why not, instead of going until -1000, just go until -20?
Like this, instead of escalating values here then there, we're tuning the values. We don't need to override a third party definition, we just need to adapt ours.

File: modules/oe_theme_content_news/oe_theme_content_news.module [hunk 1]
```diff
@@ -27,7 +27,7 @@ function oe_theme_content_news_preprocess_node__oe_news(&$variables) {
   if ($media instanceof MediaInterface) {
     $builder = \Drupal::entityTypeManager()->getViewBuilder('media');
     $variables['content']['featured_image'] = $builder->view($media, 'oe_theme_main_content');
-    // Show the image always at the top.
+    // Show the image always at the top after content_moderation_control.
```
Comment (upchuk) [inline modules/oe_theme_content_news/oe_theme_content_news.module]: No need for this change.

File: modules/oe_theme_helper/oe_theme_helper.module [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Module file additional functionality for theming.
```
Comment (upchuk) [inline modules/oe_theme_helper/oe_theme_helper.module]:
```
The OE Theme Helper module.
```

File: modules/oe_theme_helper/oe_theme_helper.module [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * Implements hook_entity_view_alter().
+ *
+ * Change content_moderation_control's weight.
+ */
+function oe_theme_entity_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  if (!isset($build['content_moderation_control'])) {
+    return;
+  }
+
+  // Set weight minor to be always on top even before News' featured_image.
```
Comment (upchuk) [inline modules/oe_theme_helper/oe_theme_helper.module]:
```
Ensure that the content moderation control block will always show at the top, above any other elements.
```

File: modules/oe_theme_helper/oe_theme_helper.module [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Implements hook_entity_view_alter().
+ *
+ * Change content_moderation_control's weight.
```
Comment (upchuk) [inline modules/oe_theme_helper/oe_theme_helper.module]: You can remove this comment.

**Pull request merged**